protected/humhub/modules/admin/controllers/GroupController.php

```diff
@@ -276,13 +276,33 @@ public function actionAdminUserSearch($keyword, $id)
 
     public function checkGroupAccess($group)
     {
-        if(!$group) {
+        if (!$group) {
             throw new HttpException(404, Yii::t('AdminModule.user', 'Group not found!'));
         }
 
-        if($group->is_admin_group && !Yii::$app->user->isAdmin()) {
+        if ($group->is_admin_group && !Yii::$app->user->isAdmin()) {
             throw new HttpException(403);
         }
     }
 
+    public function actionReassignAll($id)
+    {
+        $group = Group::findOne(['id' => $id]);
+
+        if (Yii::$app->request->isPost) {
```

Comment:
We should implement this as ActiveJob via the queue. 

Here is an example job, which can be triggered using the administration menu: 
https://github.com/humhub/humhub/blob/master/protected/humhub/modules/search/jobs/RebuildIndex.php

Reply:
Done!